recipe/meta.yaml

```diff
@@ -6,12 +6,16 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: a5bf44b1aa0adaf6d9d20f86162b34f7c593bfedabc51239953e446aefc8ce18
+  fn: uv-{{ version }}.tar.gz
+  url: https://github.com/astral-sh/ruff/releases/download/{{ version }}/source.tar.gz
+  sha256: 4f3c5d0acbefb5792ad82a903137f18e8498726dca3b678bdc3b8f8129948341
```

Comment:
Hm?

Reply:
I tried to align the recipe with uv's but looks complicated :laughing: